.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+yarn format && yarn lint
```

Comment:
`yarn format` should be enough. It exits with a non-zero code when there's code that can't be formatted